internal/providers/pke/pkeworkflow/delete_dex_client.go

```diff
@@ -48,7 +47,7 @@ func (a *DeleteDexClientActivity) Execute(ctx context.Context, input DeleteDexCl
 
 	err = a.clusterAuthService.UnRegisterCluster(ctx, cluster.GetUID())
 	if err != nil {
-		return errors.New(fmt.Sprintf("can't delete dex client for cluster %t", cluster))
+		return emperror.Wrapf(err, "can't delete dex client for cluster: %d", input.ClusterID)
```

Comment:
```suggestion
		return emperror.Wrap(err, "can't delete dex client for cluster")
```

the  cluster id should be clear from the context added during error handling

Reply:
Thanks!